index.js

```diff
@@ -76,7 +76,8 @@ class HtmlWebpackPlugin {
       // Default metaOptions if no template is provided
       if (!userOptions.template && options.templateContent === false && options.meta) {
         const defaultMeta = {
-        // From https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag
+          // TODO remove in the next major release
```

Comment:
I think we should avoid to add any meta tags in the next major release, we can provide a good default `HTML` template, it should be inside template or we should fully parse HTML document and check if we already have duplications